app/assets/javascripts/application.js

```diff
@@ -14,6 +14,7 @@
 //= require turbolinks
 //= require microlight
 //= require timeago.js/dist/timeago.min
+//= require nouislider/distribute/nouislider.min
```

Comment:
I think it would make sense to require this in `num-players-slider.js` rather than in this file.

Reply:
Makes total sense. Didn't know you could include within files beyond the `application.js`.